### PR TITLE
fix: add harshit995

### LINKED ISSUE #938: harshit995
## Profile Screenshot

SS

### PR DESCRIPTION
**Pull request by harshit995**
resolves #938 

## Checklist for profile addition

- [ ] Name the PR `fix: add <your-username>`.
- [ ] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [ ] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/harshit995.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://github.com/harshit995/harshit995/blob/main/SS.jpeg?raw=true",
```
Comment (krishguptadev) [inline profiles/harshit995.json:2]:
```suggestion
  "image": "https://user-images.githubusercontent.com/91655303/147478802-520eb460-b349-4f83-9416-25b0c8411724.png",
```

**Comment by harshit995** [comment]
@krishguptadev now you can check.....

**Review by krishguptadev — APPROVED** [review]

File: profiles/harshit995.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/91655303/147478802-520eb460-b349-4f83-9416-25b0c8411724.png",
```
Comment (Panquesito7) [inline profiles/harshit995.json:2]:
```suggestion
  "image": "https://user-images.githubusercontent.com/85628634/147473415-0a1062c9-ba5e-4602-a958-28fe8439a5f4.jpeg",
```

**Comment by harshit995** [comment]
@Panquesito7 pls check my pr

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**